{{cookiecutter.project_slug}}/tests/CI_docker/context/environment_{{cookiecutter.project_slug}}.yml

```diff
@@ -8,17 +8,30 @@ dependencies:
   - pip
 
   - pip:
-    # doc requirements
-    - coverage
+    # setup requirements
+    - bump2version
+    - wheel
+    - watchdog
+    - tox
```

Comment:
We could remove:
    - bump2version
    - wheel
    - watchdog
    
But I think `tox` we need it.

What do you think?

Reply:
That´s right but I was about to remove the requirements_dev.txt because the packages from there are not installed anyways and I can´t remember any situation where I needed them. I used to collect the dev requirements directly in the setup.py, e.g., [here](https://git.gfz-potsdam.de/danschef/arosics/-/blob/master/setup.py).

Reply:
Should we then remove `requirements.txt` and from setup the following packages:
    - bump2version
    - wheel
    - watchdog

Reply:
There is only a `requirements_dev.txt` which I would remove and instead set it up like in [geoarray](https://git.gfz-potsdam.de/danschef/geoarray).

Reply:
Sorry, it was a typo. I meant `requirements_dev.txt`. I pushed two commits to solve this conversation. 
Since you know the difference between `arosics` and `geoarray` setup, and in case I missed something, it would be better if you apply the necessary changes I might have missed.